tap_exacttarget/endpoints/folders.py

```diff
@@ -88,6 +91,9 @@ def sync_data(self):
                                      'ModifiedDate',
                                      folder.get('ModifiedDate'))
 
-            singer.write_records(table, [folder])
+            with Transformer() as transformer:
+                for rec in [folder]:
```

Comment:
Do we need to write the loop for only 1 static element? Can we write only if? for each transforms.